lib/Collaboration/Collaborators/Listener.php

```diff
@@ -118,10 +121,14 @@ protected function filterBridgeBot(array $results): array {
 	}
 
 	protected function filterExistingParticipants(string $token, array $results): array {
+		$sessionId = $this->talkSession->getSessionForRoom($token);
 		try {
-			$this->room = $this->manager->getRoomByToken($token);
+			$this->room = $this->manager->getRoomForUserByToken($token, $this->userId);
+			$this->participantService->getParticipant($this->room, $this->userId, $sessionId);
```

Comment:
The method `participantService->getParticipant` depends of this PR without backport to stable25:

https://github.com/nextcloud/spreed/pull/8106

What we will do in this case?

Reply:
Fixed with a follow up commit